### DISCUSSION TITLE
ElastAlert2 error

### DISCUSSION THREAD

**Discussion by ngms17** [jertel/elastalert2#595] [Q&A] [answered]
When running my ElastAlert2 rule, it gives me this error:

ERROR:root:Error writing alert info to Elasticsearch: RequestError(400, 'mapper_parsing_exception', "failed to parse field [match_body.user_agent.os.version] of type [long] in document with id 'SRw1gH0BNdCWR85QSq-x'. Preview of field's value: '12.0.1'")

How can i solve this?

**Comment by nsano-rururu** [comment, marked as answer]
https://github.com/Yelp/elastalert/issues/2747

**Reply by nsano-rururu** [reply]
https://github.com/Yelp/elastalert/issues/2567

**Reply by nsano-rururu** [reply]
https://github.com/Yelp/elastalert/pull/2358#commitcomment-47398802
